File: python/7 kyu/85.py
# https://www.codewars.com/kata/57aa218e72292d98d500240f/train/python

# Passed

def heron(a, b, c):
    s = (a + b + c) / 2
    area = (s * (s - a) * (s - b) * (s -c)) ** 0.5
    return area

output = heron(3, 4, 5)
print(output)

File: nodejs/7 kyu/85.js
// https://www.codewars.com/kata/5a995c2aba1bb57f660001fd/train/javascript

// Passed

function scrollingText(text) {
  text = text.toUpperCase();
  const rotatedTexts = text
    .split("")
    .map((char, idx) => text.slice(idx).concat(text.slice(0, idx)));
  return rotatedTexts;
}

const output = scrollingText("codewars");
console.log(output);